.github/workflows/deploy-preprod.yml

```diff
@@ -16,12 +16,7 @@ jobs:
       - name: Get the latest budibase release version
         id: version
         run: |
-          if [ -z "${{ github.event.inputs.version }}" ]; then
-            git pull
-            release_version=$(cat lerna.json | jq -r '.version')
-          else
-            release_version=${{ github.event.inputs.version }}
-          fi
+          release_version=$(cat lerna.json | jq -r '.version')
```

Comment:
what about rollbacks via the workflow_dispatch?

Reply:
If we use `workflow_dispatch` we will be selecting the tag we want to run on. This tag will be pulling the code that has the version in the `lerna.json`.
So, if you manually trigger the tag `v1.2.3` it will checkout the tagged code, which should have the version `1.2.3` hardcoded in there